l10n/af_ZA.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
never manually change translation files - please remove all changes to l10n folder - THX